app/src/pages/inside/settingsPage/settingsTabs/autoAnalysisTab/autoAnalysisTab.jsx

```diff
@@ -0,0 +1,126 @@
+    defaultMessage: 'Base for Auto Analysis',
+  },
+});
+@injectIntl
+export class AutoAnalysisTab extends PureComponent {
+  static propTypes = {
+    intl: intlShape.isRequired,
+    minDocFreq: PropTypes.number,
+    minShouldMatch: PropTypes.number,
+    minTermFreq: PropTypes.number,
+    numberOfLogLines: PropTypes.number,
+    indexing_running: PropTypes.bool,
+    isAutoAnalyzerEnabled: PropTypes.bool,
+    analyzer_mode: PropTypes.string,
+  };
+
+  static defaultProps = {
+    analyzer_mode: 'LAUNCH_NAME',
+    minDocFreq: 7,
+    minShouldMatch: 80,
+    minTermFreq: 1,
+    numberOfLogLines: 2,
+    indexing_running: false,
+    isAutoAnalyzerEnabled: false,
+  };
+
+  state = {
+    isSwitcherEnable: this.props.isAutoAnalyzerEnabled,
+    analyzerValue: this.props.analyzer_mode,
+  };
+
+  handleSwitcherChange = (newValue) => {
+    this.setState({
+      isSwitcherEnable: newValue,
+    });
+  };
+
+  handleRadioChange = (event) => {
+    this.setState({
+      analyzerValue: event.target.value,
+    });
+  };
+
+  render() {
+    const { intl } = this.props;
+    return (
+      <div className={cx('settings-tab-content')}>
+        <div className={cx('form-group-container')}>
+          <span className={cx('form-group-column', 'switch-auto-analysis-label')}>
+            {intl.formatMessage(messages.autoAnalysisSwitcherTitle)}
+          </span>
+          <div className={cx('form-group-column', 'switcher-wrapper')}>
+            <InputBigSwitcher
+              value={this.state.isSwitcherEnable}
+              onChange={this.handleSwitcherChange}
+            />
+          </div>
+          <div className={cx('form-group-column', 'form-group-description')}>
+            <div className={cx('form-group-help-block')}>
+              <p>If ON - analysis starts as soon as any launch finished.</p>
+              <p>If OFF - not automatic, but can be invoked manually</p>
+            </div>
+          </div>
+        </div>
+
+        <div className={cx('form-group-container')}>
+          <span
+            className={cx('form-group-column', 'switch-auto-analysis-label', 'aa-strategy-label')}
+          >
+            {intl.formatMessage(messages.strategySelectorTitle)}
+          </span>
+          <div className={cx('form-group-column', 'form-group-description')}>
+            <div className={cx('aa-strategy-option')}>
+              <div className={cx('aa-strategy-option-selector')}>
+                <InputRadio
+                  ownValue={'LAUNCH_NAME'}
+                  value={this.state.analyzerValue}
+                  onChange={this.handleRadioChange}
+                  name={'aa-strategy'}
+                >
+                  Launches with the same name
+                </InputRadio>
+              </div>
+              <span className={cx('form-group-help-block')}>
+                The test items are analyzed on base of previously investigated data in launches with
+                the same name
```

Comment:
Please use syntax like this
{intl.formatMessage(messages.autoAnalysisSwitcherTitle)}